src/components/post/create/create-post-expanded.component.tsx

```diff
@@ -37,157 +37,174 @@ type CreatePostExpandedComponentProps = {
   experiences: Experience[];
 };
 
-export const CreatePostExpandedComponent: React.FC<CreatePostExpandedComponentProps> = ({ onSubmit, text, user }) => {
-  const styles = useStyles();
-
-  const uploadImageRef = useRef<HTMLInputElement | null>(null);
-  const uploadVideoRef = useRef<HTMLInputElement | null>(null);
-  const [files, setFiles] = useState<UpoadedFile[]>([]);
-  const [postText, setPostText] = useState(text);
-  const [tags, setTags] = useState<string[]>([]);
-
-  const updatePostText = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
-    const text = event.target.value;
-
-    setPostText(text);
-  };
-
-  const selectVideo = (): void => {
-    const uploadField: any = uploadVideoRef?.current;
+export type CreatePostExpandedComponentRefProps = {
+  clearForm: () => void;
+};
 
-    if (!uploadField) return;
+export const CreatePostExpandedComponent = forwardRef(
+  ({ onSubmit, text, user }: CreatePostExpandedComponentProps, ref: React.Ref<CreatePostExpandedComponentRefProps>) => {
```

Comment:
Not  a fan of this passing of `user` around. can we put it in the global store/state?